chcemvediet/apps/accounts/admin.py

```diff
@@ -79,3 +89,20 @@ def get_queryset(self, request):
         queryset = queryset.select_related(u'user')
         queryset = queryset.select_undecided_emails_count()
         return queryset
+
+    def login_as_view(self, request, id):
+        admin_login_as = request.session.get(u'admin_login_as')
+        user = get_object_or_404(User, pk=id)
+        if not hasattr(user, u'backend'):
+            user.backend = u'chcemvediet.apps.accounts.backends.AdminLoginAsBackend'
+        login(request, user)
+        request.session[u'admin_login_as'] = admin_login_as
```

Comment:
Nerozumiem, preco najprv `admin_login_as` vyberas z `request.session` a potom tam rovnake vkladas naspat. Ved tu chces praveze nastavit `request.session[u'admin_login_as']` na ID usera, ako ktory sa admin chce prihlasit. Teda `obj_pk` parameter z URL.

Reply:
Po logine sa cela `request.session` vymazala, preto som potreboval znovu nastavit atribut. Ale uz to je nerelevantne (aj ostatne commenty). Cely PR som prerobil.